### PR TITLE
Simplify --print_config.py invocation.

### PR DESCRIPTION
**Pull request by trironkk**

### CODE REVIEW

**Comment by trironkk** [comment]
https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/issues/237

File: marketplace/deployer_util/schema_values_common.py [hunk 1]
```diff
@@ -16,31 +16,31 @@
...
+
+def add_to_argument_parser(parser):
   parser.add_argument(
       '--schema_file',
       help='Path to the schema file',
       default='/data/schema.yaml')
 
+  parser.add_argument(
+      '--mode',
+      help='"expanded" for expanded, and "raw" for not.',
```
Comment (huyhg) [inline marketplace/deployer_util/schema_values_common.py]: missing `stdin`!

File: scripts/install [hunk 2]
```diff
@@ -56,11 +56,11 @@ echo "$parameters" \
 # Extract name, namespace, and api version.
 name="$(print_config.py \
     --schema_file=/data/schema.yaml \
```
Comment (huyhg) [inline scripts/install]: we should be able to drop this?

File: scripts/install [hunk 3]
```diff
@@ -56,11 +56,11 @@ echo "$parameters" \
...
-    --values_file=/data/values.yaml \
+    --mode raw \
     --param='{"x-google-marketplace": {"type": "NAME"}}')"
 namespace="$(print_config.py \
     --schema_file=/data/schema.yaml \
```
Comment (huyhg) [inline scripts/install]: we should be able to drop this?

File: marketplace/deployer_util/schema_values_common.py [hunk 4]
```diff
@@ -16,31 +16,31 @@
...
+
+
+def add_to_argument_parser(parser):
   parser.add_argument(
       '--schema_file',
       help='Path to the schema file',
       default='/data/schema.yaml')
 
+  parser.add_argument(
+      '--mode',
```
Comment (huyhg) [inline marketplace/deployer_util/schema_values_common.py]: `values_mode` to make it clear. Especially in `provision.py`, `--mode` is very ambiguous.

File: marketplace/deployer_util/schema_values_common.py (hunk 1 above)
Comment (trironkk) [reply marketplace/deployer_util/schema_values_common.py]: Good catch!

File: marketplace/deployer_util/schema_values_common.py (hunk 4 above)
Comment (trironkk) [reply marketplace/deployer_util/schema_values_common.py]: Done!

File: scripts/install (hunk 2 above)
Comment (trironkk) [reply scripts/install]: Done!

File: scripts/install (hunk 3 above)
Comment (trironkk) [reply scripts/install]: Yup!

**Pull request merged**